### PR TITLE
(dx): Better lint

### PR DESCRIPTION
**Pull request by shahradelahi**
@MKRhere

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -5,9 +5,11 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ v4 ]
```
Comment (MKRhere) [inline .github/workflows/node.js.yml:8]: Why remove `branches`? Maybe just add v5 instead?

Comment (shahradelahi) [reply .github/workflows/node.js.yml:8]: By removing `branches`, it's going to run on any push event on any branch. Why just `v5`? Are you going to drop support for `v4`?

Comment (MKRhere) [reply .github/workflows/node.js.yml:8]: No, I mean add v5 to the existing array and make it `[v4, v5]`. It also runs on pull requests, so that's fine.

**Review by MKRhere — APPROVED** [review]
lgtm

**Review by the1Riddle — COMMENTED** [review]
👍

**Pull request merged**